include/searcher.h

```diff
@@ -85,12 +85,12 @@ class Searcher
    *
    * @param search The search query.
    * @param resultStart the start offset of the search results (used for pagination).
-   * @param resultEnd the end offset of the search results (used for pagination).
+   * @param resultMax Maximum results to get from start (used for pagination).
    * @param verbose print some info on stdout if true.
    */
   void search(const std::string& search,
               unsigned int resultStart,
-              unsigned int resultEnd,
+              unsigned int resultMax,
```

Comment:
Maybe rename this to `maxResults` or `resultCount`.

`resultMax` make me think about the "result maximum (highest)". So a single result, which is semantically wrong.

Reply:
or `resultMaxCount`

Reply:
renamed to `maxResultCount`